src/data-tutorials/users.tsx

```diff
@@ -894,6 +894,13 @@ const Users: User[] = [
     website: 'blog/tutorials/3-simple-steps-to-get-started-with-valora-on-celo',
     tags: ['beginner','foundation', 'valora'],
   },
+  {
+    title: 'All you need to know about Celo to Ease your Web3 Development Journey you use Celo Blockchain',
+    description: 'In this tutorial, we will introduce you to the Celo blockchain and explain the key features and benefits of using it for your web3 development projects. We will cover the basics of Celo and provide an overview of its technology and ecosystem, as well as discuss the tools and resources available to help you get started. By the end of this tutorial, you will have a better understanding of Celo and how it can support your web3 development journey. Whether you are just starting out or are an experienced developer, this tutorial will provide you with the knowledge you need to get started with Celo',
+    preview: require('./showcase/beginner/All-you-need-to-know-about-Celo-to-Ease-your-Web3-Development.png'),
+    website: '/blog/tutorials/All-you-need-to-know-about-Celo-to-Ease-your-Web3-Development-Journey',
+    tags: ['celosage','celo','video','remote'], 
```

Comment:
Use the advised tags, e.g. celo, celo sage, beginner

Reply:
giving me error